wazo_dird_plugin_odoo/plugin.py

```diff
@@ -19,89 +19,39 @@ class OdooView(BaseBackendView):
     list_resource = http.OdooList
     item_resource = http.OdooItem
 
+class OdooPlugin(BaseServicePlugin):
 
-class OdooPlugin(BaseSourcePlugin):
+    def name(self):
+        return 'Odoo backend'
 
-    def load(self, dependencies):
-        config = dependencies['config']
+    def load(self, config):
+        self.db = config['config']['odoo_config']['database']
```

Comment:
I don't think this works. Have you tried with master, i made lot of changes to make it works with wazo-platform to configure the plugin.